app/models/user.rb

```diff
@@ -76,7 +76,10 @@ def password_reset_expired?
   end
 
   def feed
-    microposts
+    following_ids = "SELECT followed_id FROM relationships
+                    WHERE follower_id = :user_id"
+    Micropost.where("user_id IN (#{following_ids})
```

Comment:
please use scope
in micropost.rb
```suggestion
   scope :by_user, ->(ids){where user_id: ids}
```
and call it here
Micropost.by_user following_ids << id

Reply:
please fix it before ready

Reply:
call it here
Micropost.by_user following_ids << id